modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -90,4 +102,22 @@ def run_host(ip)
       end
     end
   end
+
+  def set_sane_defaults
```

Comment:
This can be nuked and put into the mssql mixin which will know how to instantiate the client directly